app/lab/package.json

```diff
@@ -184,6 +184,7 @@
     "whatwg-fetch": "^3.0.0"
   },
   "jupyterlab": {
+    "name": "lab",
```

Comment:
Perhaps we should we put it in `jupyterlite`? Doesn't look like it appears [upstream](https://github.com/jupyterlab/jupyterlab/blob/master/builder/metadata_schema.json)....

Reply:
It's not in the schema upstream, but can be found in `dev_mode/package.json`: https://github.com/jupyterlab/jupyterlab/blob/fc73254325aaf445678c411d53e736d3027f81a2/dev_mode/package.json#L209